pygsheets/cell.py

```diff
@@ -171,6 +173,21 @@ def set_text_format(self, attribute, value):
         self.update()
         return self
 
+    def set_text_rotation(self, attribute="angle", value=0):
+        """
+        set the text rotation
+        :param attribute: "angle"
+        :param value: corresponding value for the attribute
+        :return: :class: Cell
+        """
+        if attribute != "angle":
```

Comment:
As per [docs](https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets#textrotation) `vertical` is also an possible param. Also there is not point in asking attribute as a param if its the only attribute user can set.